tests/testthat/test-truncate_linelist.R

```diff
@@ -3,44 +3,51 @@ ll <- sim_linelist()
 
 test_that("truncate_linelist works as expected with defaults", {
   ll_trunc <- truncate_linelist(ll)
-  # in this example dataset only the outcome col is adjusted from truncate_linelist
+  # in this example dataset one case is removed
   expect_false(identical(ll$date_outcome, ll_trunc$date_outcome))
+  expect_gt(nrow(ll), nrow(ll_trunc))
+})
+
+test_that("truncate_linelist works as expected with modified truncation_time", {
+  ll_trunc <- truncate_linelist(
+    linelist = ll,
+    truncation_time = 60
+  )
+  expect_gt(nrow(ll), nrow(ll_trunc))
```

Comment:
this is just testing that some rows have been cut - might be worth checking the right thing has been cut?

Reply:
More exact expectations in tests implemented in a58d77cc46c093744bcff681f591f980581b366e.